### DISCUSSION TITLE
Calendar component weekview functionality

### DISCUSSION THREAD

**Discussion by Pamposgsk** [tailwindlabs/tailwindcss#8481] [Help]
Hi,
Is there a possibility to drag n drop external events to the calendar component in weekview.
Furthermore can I have a link in order to interact with this component before buy?
Thanks

**Comment by Farr69** [comment]
Show a week calendar instead of a monthly calendar view to save space. By setting the calendarType to 'week' and passing the count in the weeks option you can simply enable a week view. You can dynamically change the number of weeks or switch between month and week view without the need for recycling the whole component.
[JCPenney Kiosk Associates](https://www.jcpenneykiosk.review/)
